cdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to MongoDB Atlas CDK
```

Comment:
Getting the contributing guides split out ahead of our planning repo split for cdk